GOVERNANCE.md

```diff
@@ -3,43 +3,58 @@ the Benevolent dictator governance model by Ross
 
-Users are encouraged to participate in the life of the project and the community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user activities include (but are not limited to):
+While core developers can modify code directly, this ability is rarely used. Instead, changes are proposed as pull requests, and are only merged after they have been reviewed by at least one other core developer. Changes to the API (especially breaking changes) must also be approved by the project lead. 
 
-- evangelising about the project
-- informing developers of project strengths and weaknesses from a new user’s perspective
-- providing moral support (a ‘thank you’ goes a long way)
+Core developers are recruited from contributors. An invitation to join the core team can be extended to anyone who has made a major contribution, either through a small number of large changes, or a consistent pattern of smaller contributions. Any existing core developer can propose a contributor be invited to the core team by emailing the project lead. The project lead will the confirm the invitation with existing core developers.
 
-Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described above.
+The core developers of ggplot2 are:
+
+* [Winston Chang](https://github.com/wch)
+* [Lionel Henry](https://github.com/lionel-)
+* [Thomas Lin Pedersen](https://github.com/thomasp85)
+* [Kohske Takahashi](https://github.com/kohske)
+* [Claus Wilke](https://github.com/clauswilke)
+* [Kara Woo](https://github.com/karawoo)
+
+All core developers are bound by the [code of conduct](CODE_OF_CONDUCT.md).
+
+## Project lead
+
+The project lead, [Hadley Wickham](http://github.com/hadley), is responsible for:
+
+* Setting, and clearly communication the strategic objectives of the project.
+* Mediating any conflicts amongst the core developers.
+* Ensuring that the project survives in the long term.
+
+The project lead is bound by the [code of conduct](CODE_OF_CONDUCT.md).
+
+# Decision-making process
 
-# Decision-Making Process
-This project makes decisions according to a consensus model where suggestions
-are considered and discussed between the community and core developers. In case 
-of conflict, the project lead’s word is final. If the community chooses to question
-the wisdom of the actions of a core developer, the project lead can review their
-decision, and either uphold or reverse them.
+This project makes decisions according to a consensus model where suggestions are considered and discussed between the community and core developers. In case of conflict, the project lead’s word is final. If the community chooses to question the wisdom of the actions of a core developer, the project lead can review their decision, and either uphold or reverse them.
```

Comment:
This project makes decisions by consensus: suggestions are considered and discussed by the community and core developers. [question: where does discussion take place? in issue threads? mailing list?] Where consensus cannot be reached, the project lead's word is final.  If the community questions a decision, the project lead may review it and either uphold or reverse it.